docs/jsdoc/options.jsdoc

```diff
@@ -36,6 +36,10 @@
  * The filename of the template. Required for inclusion and caching unless 
  * you are using {@link module:ejs.renderFile}. Also used for error reporting.
  * 
+ * @property {String}  [root='undefined']
```

Comment:
OK @cnwhy, remove the quotes around `undefined` so the line looks like this:
```
 * @property {String}  [root=undefined]
```

The `filename` wording is incorrect; I will fix that.

Sorry for the long wait.